From 6735cbbdfd7cb5a55d01cb411bf5f61dde8946bb Mon Sep 17 00:00:00 2001
From: ben <benvansleen@gmail.com>
Date: Wed, 28 Jan 2026 18:26:42 -0500
Subject: [PATCH 2/2] point to benvansleen reedline multiple key combo branch

---
 Cargo.lock | 5 ++---
 Cargo.toml | 2 +-
 2 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/Cargo.lock b/Cargo.lock
index ade96ef86..227c9a064 100644
--- a/Cargo.lock
+++ b/Cargo.lock
@@ -6565,9 +6565,8 @@ dependencies = [
 
 [[package]]
 name = "reedline"
-version = "0.44.0"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "0fb6657e89284163d4c081738148bd3c2c7487586898b38a004c3ea432c1a6f3"
+version = "0.45.0"
+source = "git+https://github.com/benvansleen/reedline.git?branch=feat%2Fmultiple-key-combo-support#2ddeaa0395236e721638c50e73b33f0b6de36ab4"
 dependencies = [
  "arboard",
  "chrono",
diff --git a/Cargo.toml b/Cargo.toml
index a2442a981..2faecc0d4 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -154,7 +154,7 @@ getrandom = "0.2" # pick same version that rand requires
 rand_chacha = "0.9"
 ratatui = "0.29"
 rayon = "1.11"
-reedline = "0.44.0"
+reedline = { git = "https://github.com/benvansleen/reedline.git", branch = "feat/multiple-key-combo-support" }
 rmp = "0.8"
 rmp-serde = "1.3"
 roxmltree = "0.20"
-- 
2.52.0
