From 5efe69cdf77427ded1e5eae69813349501bfbc9c Mon Sep 17 00:00:00 2001
From: ben <benvansleen@gmail.com>
Date: Sun, 29 Jun 2025 14:31:36 -0400
Subject: [PATCH] patch cargo.toml

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 0efcc2e58..4316afe27 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -140,7 +140,7 @@ getrandom = "0.2" # pick same version that rand requires
 rand_chacha = "0.9"
 ratatui = "0.29"
 rayon = "1.10"
-reedline = "0.40.0"
+reedline = { git = "https://github.com/benvansleen/reedline" }
 rmp = "0.8"
 rmp-serde = "1.3"
 roxmltree = "0.20"
-- 
2.49.0

